src/app/containers/SocialEmbed/index.jsx

```diff
@@ -50,10 +52,9 @@ const SocialEmbedContainer = ({ blocks }) => {
   if (!href) return null;
 
   const oEmbed = path(['embed', 'oembed'], model);
-  const maxHeight = path(
-    ['embed', 'fallback_image', 'fallback_image_height'],
-    model,
-  );
+  const maxHeight = path(['height'], oEmbed);
+
+  const lazyLoadHeight = maxHeight || MIN_HEIGHT;
```

Comment:
This looks like a good solution!
- You could rename `maxHeight` to `height` and use pathOr instead of `||` to set the default of `MIN_HEIGHT` if you wanted to reduce complexity.

Reply:
yep, that's what I originally did but needed to a way to easily write this conditional 
```
    maxHeight ? `max-height: ${maxHeight}px;` : `min-height: ${MIN_HEIGHT}px`}
```

Reply:
if I set defaults height would always be  defined.